python/cudf/cudf/core/column/categorical.py

```diff
@@ -95,35 +84,59 @@ class CategoricalColumn(column.ColumnBase):
         "__gt__",
         "__ge__",
     }
+    # TODO: See if we can narrow these integer types
+    _VALID_PLC_TYPES = {
```

Comment:
Where are these sets used?

Reply:
They are used in `ColumnBase.__init__`. Subclasses override which `plc.TypeId`s they expect the pylibcudf Column to have and the base class checks `plc_column.type().id() in self._VALID_PLC_TYPES`